test/helpers/wrappers.go

```diff
@@ -129,6 +130,20 @@ func CurlWithRetries(endpoint string, retries int, fail bool, optionalValues ...
 		retries, endpoint)
 }
 
+// CurlTimeout does the same as CurlFail() except you can define the timeout.
+// See note about optionalValues on CurlFail().
```

Comment:
Sounds like a missed opportunity for some refactoring ;)

Reply:
Indeed, I'll refactor this in an separate commit when rebasing after node2node has been merged, if this is ok.